src/tests/ftest/aggregation/dfuse_space_check.py

```diff
@@ -89,52 +96,49 @@ def test_dfusespacecheck(self):
         self.create_pool()
         self.create_cont()
         self.start_dfuse(self.hostlist_clients, self.pool, self.container)
 
-        # get scm space before write
-        self.space_before = self.get_nvme_free_space()
+        # get nvme space before write
+        self.initial_space = self.get_nvme_free_space()
 
-        # create large file and perform write to it so that if goes out of
-        # space.
-        large_file = str(self.dfuse.mount_dir.value + "/" + "largefile.txt")
-        cmd = "touch {}".format(large_file)
-        self.execute_cmd(cmd)
-        dd_count = ((self.space_before / self.block_size) + 1)
+        # Create a file as large as we can
+        large_file = os.path.join(self.dfuse.mount_dir.value, 'largefile.txt')
+        self.execute_cmd('touch {}'.format(large_file))
+        dd_count = (self.initial_space // self.block_size) + 1
         write_dd_cmd = "dd if=/dev/zero of={} bs={} count={}".format(
             large_file, self.block_size, dd_count)
         self.execute_cmd(write_dd_cmd, False)
 
-        # store free space after write and remove the file
-        rm_large_file = "rm -rf {}".format(large_file)
-        self.execute_cmd(rm_large_file)
+        # Remove the file
+        self.execute_cmd('rm -rf {}'.format(large_file))
 
-        # Check if aggregation is complete.
-        self.check_aggregation()
+        # Wait for aggregation to complete
+        self.wait_for_aggregation()
 
-        # Once aggregation is complete, write multiple files of small size
-        # until pool is out of space and store how many files were created.
+        # Write small files until we run out of space
         file_count1 = self.write_multiple_files()
 
         # remove all the small files created above.
-        self.execute_cmd("rm -rf {}/*".format(self.dfuse.mount_dir.value))
+        self.execute_cmd("rm -rf {}".format(os.path.join(self.dfuse.mount_dir.value, '*')))
 
-        # Check for aggregation to complete after file removal.
-        self.check_aggregation()
+        # Wait for aggregation to complete after file removal
+        self.wait_for_aggregation()
 
-        # Write small sized multiple files again until pool is out of space
-        # and store the number of files created.
+        # Write small files again until we run out of space
+        # and verify that we wrote at least as many as file_count1
         file_count2 = self.write_multiple_files()
-
-        # Check if both the files counts is equal. If not, fail the test.
-        if file_count1 != file_count2:
-            self.fail("Space was not returned completely after re-writing the "
-                      "same number of files after deletion")
+        self.log.info('file_count1 = %s\nfile_count2 = %s', file_count1, file_count2)
+        self.assertGreaterEqual(
+            file_count2, file_count1,
+            'Space was not returned. Expected to write at least {} files'.format(file_count1))
```

Comment:
(pylint-consider-using-f-string) Formatting a regular string which could be a f-string